crafting/builder

```diff
@@ -3,5 +3,5 @@ Builder- (5)
 -- Wall - 12 parts - Protects all members of their team from all attacks for once. Structure will remain if not attacked.
 -- Fort - 21 parts - Protects all members of their team from all attacks, twice. Structure will remain if not attacked.
 -- Spikes - 7 parts - The builder can choose to add spiked to any already existing structures, which will kill the first attacker who tries to attack the structure. Note that spikes do not stop the structure from breaking. The attacks will still break the structures. Any kills using the spiked do not award any parts.
-- The builder continues to get parts even when dead. However, they can only build when alive. Structures take 1 night to get built and only get built after the attacks occur. Two structures cannot exist at once. All of builder's protections are strong protections.
+- The builder continues to get parts even when dead. However, they can only build when alive. Structures are built as an end night effect. Two structures cannot exist at once. All of builder's protections are strong protections.
```

Comment:
Wait if it's end night does it mean it happens before the attacks? So the structures protect for the same night as they are made?

Reply:
oh sorry i forgot to change it to post-night action, my bad for not going back and fixing it

Reply:
Oh alright nvm